<launch>
<node pkg="explore_lite" type="explore" respawn="false" name="explore">
  <param name="robot_base_frame" value="base_link"/>
  <param name="costmap_topic" value="proj_map"/>
  <param name="costmap_updates_topic" value="map_updates"/>
  <param name="visualize" value="true"/>
  <param name="planner_frequency" value="1.0"/>
  <param name="progress_timeout" value="30.0"/>
  <param name="potential_scale" value="3.0"/>
  <param name="orientation_scale" value="10.0"/>
  <param name="gain_scale" value="0.1"/>
  <param name="transform_tolerance" value="0.3"/>
  <param name="min_frontier_size" value="0.25"/>
  <param name="robot_base_frame" value="base_link"/>
  <param name="map_compression" value="2"/>
  <param name="expand_obstacles" value="1"/>
</node>

<node pkg="theta_star_planner" type="theta_star_planner_node" name="planner">
  <param name="agent_radius" value="0.01"/>
  <remap from="/map" to="/projmap_resized"/>
</node>

<node pkg="tx2_action_server" type="tx2_action_server_external_driver.py" name="action_server" output="screen">
  <param name="tolerance" value="1"/>
  <param name="rate" value="5"/>
  <param name="max_path_fails" value="5"/>
</node>
</launch>
